content/practices/lean-experiments/index.md

```diff
@@ -0,0 +1,173 @@
+   {{< /callout >}}
+
+1. **Catalog the team’s “leap-of-faith” assumptions**
+   
+   Add this term to the whiteboard, including the definition so that the team can refer to it throughout the exercise (Assumptions that if we’re wrong about, our product will likely fail).
+   
+   Ask the team to brainstorm leap-of-faith assumptions on sticky notes for 2 minutes. If a team member comes up with more than two, ask them to self-edit down to those that are most important.
+   
+   Catalog the product’s unvalidated, leap-of-faith assumptions on the whiteboard.
+   
+   {{< callout >}}
+   Tip: Try framing assumptions in the positive (users will feel safe getting into a stranger’s car) rather than the negative (no one will want to get in a stranger’s car). This will make it easier to reflect on how much evidence we have/need.
+   {{< /callout >}}
+   
+   If you end up with more than a few leap-of-faith assumptions, prioritize one to focus on first. Use the [Assumptions](/practices/assumptions) practice or consider a 2x2 with “more likely to stop the project” vs. “less likely to stop the project” on the Y-axis and “needs more evidence” vs. “have lots of evidence” on the X-axis. High likelihood of stopping the project assumptions that need more evidence are the target.
+   
+1. **Explain that an assumption needs to be converted into a falsifiable hypothesis:**
+   
+   > ***[Specific testable action]*** will drive ***[expected measurable outcome]***
+   
+   Introduce the format/template that you will use to track your hypotheses, experiments and measurements. 
+   
+   For example:
+   
+   - Lean Experiment Template ***NOTE: THIS WAS A LINK TO A PIVOTAL BRANDED INTERNAL PDF***
+   - [Strategyzer Test Card](https://assets.strategyzer.com/assets/resources/the-test-card.pdf)
+   
+   Describe what each of the following represent:
+   
+   - Hypothesis — The thing we believe to be true
+   - Experiment — The test that we will run to validate our hypothesis
+   - Measurement — What signal we will look to for validation
+   - Outcome — The success and failure criteria
+   
+   {{< callout >}}
+   Tip: Don’t forget to timebox your experiments. The length of time you’ll have the experiment running should be defined up front.
+   {{< /callout >}}
+   
+   {{< callout >}}
+   Tip: If your team is not familiar with what makes good metrics, consider giving a primer or examples
+   {{< /callout >}}
+
+1. **Generate experiments**
+   
+   Distribute a sheet of paper to each team member and instruct them to:
+   
+   Copy the format of the Hypothesis & Experiment template
+   Write the leap-of-faith assumption as a falsifiable hypothesis
+   If there are more than 4 participants, consider breaking into teams to generate experiments
+   
+   {{< callout >}}
+   Tip: Print out your templates, email/share them, or create and share the digital workspace before the workshop. 
```

Comment:
Copy the lean experiment template from this miro board: https://miro.com/app/board/o9J_lGThxtc=/

Reply:
Updated